application/view/omeka/admin/item/sidebar-select.phtml

```diff
@@ -64,13 +64,18 @@ $expanded = $resourceClassId || $itemSetId || $id;
     </div>
 
     <?php if ($itemsFound): ?>
-    <button type="button" class="quick-select-toggle"><?php echo $translate('Quick add'); ?></button>
+    <span class="sr-only quick-select-on success" tabindex="-1"><?php echo $translate('Quick add checkboxes are now available.'); ?></span>
```

Comment:
It seems to me that these two spans are always screenreader-visible, am I right on that? Seems like they could get read outside of being explicitly focused by the JS.

Reply:
That's a good point, as I was only testing via tabbing through. They should probably have `display: none` when not active.